arviz/stats/stats.py

```diff
@@ -1117,7 +1123,7 @@ def summary(
         hpd_lower, hpd_higher = xr.apply_ufunc(
             _make_ufunc(hpd, n_output=2),
```

Comment:
I think this is no longer necessary, `hpd` uses `apply_ufunc` and should work straight away, I am actually quite amazed this still works. I'll post an issue